cmd/minikube/cmd/start.go

```diff
@@ -520,6 +523,9 @@ func selectDriver(existing *config.ClusterConfig) (registry.DriverState, []regis
 	if d := viper.GetString("vm-driver"); d != "" {
 		ds := driver.Status(viper.GetString("vm-driver"))
 		if ds.Name == "" {
+			if d == "kvm" {
+				d = "kvm2"
```

Comment:
I think it is safer to also use viper.Set... (to set it globally, other parts of the code I think they might get the driver from the Viper.

Reply:
@medyagh  please verify the below code. is it fine?
```

if d == "kvm" {
        viper.Set("driver","kvm2")
	d = "kvm2"
}
```

Reply:
yes